stdcommands/catfact/catfacts.go

```diff
@@ -114,8 +114,8 @@ var Catfacts = []string{
 	`There are more than 500 million domestic cats in the world, with approximately 40 recognized breeds.`,
 	`Approximately 24 cat skins can make a coat.`,
 	`While it is commonly thought that the ancient Egyptians were the first to domesticate cats, the oldest known pet cat was recently found in a 9,500-year-old grave on the Mediterranean island of Cyprus. This grave predates early Egyptian art depicting cats by 4,000 years or more.`,
-	`During the time of the Spanish Inquisition, Pope Innocent VIII condemned cats as evil and thousands of cats were burned. Unfortunately, the widespread killing of cats led to an explosion of the rat population, which exacerbated the effects of the Black Death.`,
-	`During the Middle Ages, cats were associated with withcraft, and on St. John’s Day, people all over Europe would stuff them into sacks and toss the cats into bonfires. On holy days, people celebrated by tossing cats from church towers.`,
+	`CW // animal cruelty/death ||During the time of the Spanish Inquisition, Pope Innocent VIII condemned cats as evil and thousands of cats were burned. Unfortunately, the widespread killing of cats led to an explosion of the rat population, which exacerbated the effects of the Black Death.||`,
+	`CW // animal cruelty/death ||During the Middle Ages, cats were associated with withcraft, and on St. John’s Day, people all over Europe would stuff them into sacks and toss the cats into bonfires. On holy days, people celebrated by tossing cats from church towers.||`,
```

Comment:
```suggestion
	`Content warning // animal cruelty/death ||During the time of the Spanish Inquisition, Pope Innocent VIII condemned cats as evil and thousands of cats were burned. Unfortunately, the widespread killing of cats led to an explosion of the rat population, which exacerbated the effects of the Black Death.||`,
	`Content warning // animal cruelty/death ||During the Middle Ages, cats were associated with withcraft, and on St. John’s Day, people all over Europe would stuff them into sacks and toss the cats into bonfires. On holy days, people celebrated by tossing cats from church towers.||`,
```

Reply:
Sounds good. I learned this abbreviation rather recently myself, so good idea.

Reply:
Prime example 😄
Heard today for the first time about this, as well.